travis/scripts/05-run.sh

```diff
@@ -32,11 +32,11 @@ launchCurlOrProtractor() {
 
     if [ "$status" -ne 0 ]; then
         echo "[$(date)] Not connected after" $retryCount " retries."
-        exit 1
+        return 1
```

Comment:
we should return a code to the calling function and not stop the whole script to give it a chance to kill the process